src/labels_transfer/utils/helper.py

```diff
@@ -0,0 +1,47 @@
+
+def _setup_logger():
```

Comment:
I would move it to general utils. Many scripts use this or similar code

Reply:
Ok!